server/app/models/grid_service.rb

```diff
@@ -67,8 +67,12 @@ def to_path
   end
 
   # @return [String]
-  def name_with_stack
-    "#{self.stack.name}-#{self.name}"
+  def name_with_stack(ignore_default_stack: false, stack_separator: '-')
```

Comment:
What if we add this functionality to `to_path` method? I agree with @SpComb that now it's a bit confusing.

`def to_path(ignore_grid: false)`